src/lib.rs

```diff
@@ -255,9 +259,11 @@ mod dl {
 mod dl {
     use std::ffi::OsStr;
     use std::iter::Iterator;
-    use std::libc::consts::os::extra::ERROR_CALL_NOT_IMPLEMENTED;
+    use winapi::winerror::ERROR_CALL_NOT_IMPLEMENTED;
+    use winapi;
+    use winapi::minwindef;
```

Comment:
Everything in winapi is glob imported into the root of the crate, so there's no need to refer to the submodules.